src/main/resources/permissions/mod-tlr.csv

```diff
@@ -15,3 +15,5 @@ dcb.ecs-request.transactions.post
 circulation.requests.allowed-service-points.get
 dcb.transactions.get
 dcb.transactions.put
+consortia.publications.item.post
+circulation.settings.item.post
```

Comment:
Why are we granting permissions to system user? I believe nothing we do in this PR is done though system user

Reply:
reverted